releases/cockroachcloud-01222021.md

```diff
@@ -0,0 +1,20 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+---
+
+## January 22, 2020
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### General Changes
+
+New CockroachCloud clusters will now run CockroachDB [v20.2.3](v20.2.3.html).
+
+### Bug Fixes
+
+- Fixed a bug where deleting your only organization would prevent your email from being used for a new organization in the future.
+- Fixed a bug where VPC peering appeared to be available on clusters that it wasn't supported on.
```

Comment:
Only feedback is that maybe "VPC peering" can be a link to some docs we have on VPC peering? (if we have)

Reply:
Done